spec/chewy/rake_helper_spec.rb

```diff
@@ -393,4 +393,37 @@
       OUTPUT
     end
   end
+
+  describe '.reindex' do
+    before do
+      journal
+      CitiesIndex.create!
+      CountriesIndex.create!
+    end
+
+    let(:source_index) { 'cities' }
+    let(:dest_index) { 'countries' }
+    let(:indexes_array) { [source_index, dest_index] }
+
+    xcontext 'with right arguments' do
```

Comment:
The skipping of the tests is a leftover, right?
```suggestion
    context 'with correct arguments' do
```

Reply:
Yes, I removed the skipping.